configs/openSUSE/cron-whitelist.toml

```diff
@@ -3,207 +3,155 @@ package = "rpmlint-integration-test"
 type = "cron"
 note = "valid test whitelisting entry for the OBS integration test package"
 bug = "bsc#1188704"
-digests = [
-    {
-        path = "/etc/cron.hourly/rpmlint_cron",
-        hash = "3cf3c05e5c37b0ee7b793f9c5c9b57b74071b02137a7e66a10a52d91e0ddbc51"
-    }
-]
+[[FileDigestGroup.digests]]
+path = "/etc/cron.hourly/rpmlint_cron"
+hash = "3cf3c05e5c37b0ee7b793f9c5c9b57b74071b02137a7e66a10a52d91e0ddbc51"
 
 [[FileDigestGroup]]
 package = "rpmlint-integration-test"
 type = "cron"
 note = "mismatching test whitelisting entry for the OBS integration test package"
 bug = "bsc#1188704"
-digests = [
-    {
-        path = "/etc/cron.hourly/rpmlint_mismatch",
-        hash = "4a5e8e9ca514c73f1dbee55e88c4d75a235473c1d5cef4751d84016833961fdf"
-    }
-]
+[[FileDigestGroup.digests]]
+path = "/etc/cron.hourly/rpmlint_mismatch"
+hash = "4a5e8e9ca514c73f1dbee55e88c4d75a235473c1d5cef4751d84016833961fdf"
 
 [[FileDigestGroup]]
 package = "cronie-anacron"
 type = "cron"
 note = "Executes daily, weekly, monthly cron jobs"
 bug = "bsc#1150541"
-digests = [
-    {
-        path = "/etc/cron.hourly/0anacron",
-        hash = "aa129d2165f669770b20d20fe5d826f242a069a8f9fc2323333b91d0c9ca40c9"
-    },
-]
+[[FileDigestGroup.digests]]
+path = "/etc/cron.hourly/0anacron"
+hash = "aa129d2165f669770b20d20fe5d826f242a069a8f9fc2323333b91d0c9ca40c9"
 
 [[FileDigestGroup]]
 package = "cronie-anacron"
 type = "cron"
 note = "Minor change in some logic that checks whether the system runs on battery"
 bug = "bsc#1190521"
-digests = [
-    {
-        path = "/etc/cron.hourly/0anacron",
-        hash = "6e8a152a16e84ddc10e8ab1c2ed2bad28adbfc3b0b1ced62518c4ab0ada87220"
-    },
-]
+[[FileDigestGroup.digests]]
+path = "/etc/cron.hourly/0anacron"
+hash = "6e8a152a16e84ddc10e8ab1c2ed2bad28adbfc3b0b1ced62518c4ab0ada87220"
 
 [[FileDigestGroup]]
 package = "logdigest"
 type = "cron"
 note = "scours through log files to find 'interesting' information and mails it to root"
 bug = "bsc#1150546"
-digests = [
-    {
-        path = "/etc/cron.daily/logdigest",
-        hash = "1dcd29ceaa35914c126bf76b0b00b955de332744636573dc2b907c1c542f81c4"
-    },
-    {
-        path = "/etc/cron.d/logdigest-all-ignores-autogeneration",
-        hash = "052d8f93980ade99937c2a4b972d5cf5a77367c9af6c7b02f0e05e8ca5b06797"
-    },
-]
+[[FileDigestGroup.digests]]
```

Comment:
How exactly is the coupling between the logdigest package and the digest established? Only by way of the order in which the sections appear?

Reply:
Please take a look at the specs here: https://toml.io/en/v1.0.0#table

Reply:
Okay then to answer my own question by way of the spec:

```
Any reference to an array of tables points to the most recently defined table element of the array. This allows you to define sub-tables, and even sub-arrays of tables, inside the most recent table.
```

So my assumption is correct.

I find this TOML table syntax confusing though, not really intuitive.

Reply:
Yeah, fully understand your concern, but it's what TOML is about.